src/azul/deployment.py

```diff
@@ -85,20 +85,48 @@ def api_gateway_export(self, gateway_id):
         return json.load(response['body'])
 
     @property
-    def es_endpoint(self) -> Netloc:
+    def es_endpoint(self) -> Optional[Netloc]:
```

Comment:
This method and the one below are 95% identical. We should extract the common parts into a method. This should have occurred to either of us earlier.